src/py/migraphx_py.cpp

```diff
@@ -152,13 +152,34 @@ struct npy_format_descriptor<migraphx::fp8::fp8e4m3fnuz>
 {
     static std::string format()
     {
-        // following: https://docs.python.org/3/library/struct.html#format-characters
-        // TODO: need to figure out correct encoding
+        // TODO: no standard format in numpy for fp8
         return "z";
```

Comment:
Do we not want to have a warning here as well?

Reply:
Not so much a warning as numpy straight up has no way to parse fp8. https://github.com/ROCm/AMDMIGraphX/issues/3422

Reply:
All good. Then we should be explicit about what encoding we're using somewhere and have a link in our docs somewhere to be clear what we're using.